worldedit-core/src/main/java/com/sk89q/worldedit/entity/Player.java

```diff
@@ -423,7 +423,7 @@ default void unregister() {
         if (Settings.settings().CLIPBOARD.USE_DISK && Settings.settings().CLIPBOARD.DELETE_ON_LOGOUT) {
             session.deleteClipboardOnDisk();
         } else if (Settings.settings().CLIPBOARD.USE_DISK) {
-            Fawe.instance().getClipboardExecutor().submit(getUniqueId(), () -> session.setClipboard(null));
+            Fawe.instance().submitUUIDKeyQueuedTask(getUniqueId(), () -> session.setClipboard(null));
```

Comment:
Kinda the same as my first comment, `#setClipboard` has basically no work except for synchronizing on the lock - is enqueuing that worth it? Might be required due to the lock holder thread, but that would require `getClipboard` calls to be enqueued as well, which they aren't.

Reply:
Yeah so basically "free" synchronization from the main thread without having to wait for any potentially ongoing clipboard loading (players joining and leaving a server instantaneously could otherwise lag the server)